cfme/tests/infrastructure/test_set_default_filter.py

```diff
@@ -0,0 +1,24 @@
+import pytest
+
+from cfme.fixtures import pytest_selenium as sel
+from utils.providers import setup_a_provider
+from cfme.web_ui import listaccordion as list_acc
+
+
+@pytest.fixture(scope="module")
+def provider():
+    return setup_a_provider(prov_class="infra", prov_type="rhevm")
+
+
+def test_set_default_filter_host(provider):
+    sel.force_navigate('infrastructure_hosts')
+    list_acc.select('Filters', 'Apply this filter')
+    sec_default_btn = "//div[@id='accordion']/a[contains(normalize-space(text()), 'Set Default')]"
```

Comment:
This should live somewhere outside of the test scope. At least at the module level but preferably somewhere in the models where it makes sense.